src/stores/Frame/FrameStore.ts

```diff
@@ -1180,7 +1180,7 @@ export class FrameStore {
         this.moving = false;
         this.zooming = false;
         this.colorbarLabelCustomText = this.requiredUnit === undefined || !this.requiredUnit.length ? "arbitrary units" : this.requiredUnit;
-        this.titleCustomText = "";
+        this.titleCustomText = frameInfo.fileInfo.name;
```

Comment:
add optional chaining: `frameInfo?.fileInfo?.name`. This will ensure unit tests don't break.

Reply:
fixed